mne/utils/check.py

```diff
@@ -432,3 +432,40 @@ def _check_one_ch_type(info, picks, noise_cov, method):
     elif method == 'dics' and sum(ch_types) > 1:
         warn('The use of several sensor types with the DICS beamformer is '
              'not heavily tested yet.')
+
+
+def _check_option(parameter, value, allowed_values, error=ValueError):
+    """Check the value of a parameter against a list of valid options.
+
+    Raises an exception with a readable error message if the value was invalid.
+
+    Parameters
+    ----------
+    parameter : str
+        The name of the parameter to check. This is used in the error message.
+    value : str
+        The value of the parameter to check.
+    allowed_values : list of str
+        The list of allowed values for the parameter.
+    error : type of Exception
+        The error type to raise if the value was invalid. Defaults to
+        ``ValueError``.
+
+    Returns
+    -------
+    is_valid : bool
+        Whether the value was valid or not.
```

Comment:
if its not valid, you are not returning.

Reply:
Good point.

Reply:
I would remove the "Returns" section as it returns None or raises a ValueError.

besides LGTM

Reply:
@wmvanvliet this last one please